RELEASE.md

```diff
@@ -27,7 +27,7 @@ The official binaries for KUDO are created using [goreleaser](https://goreleaser
 1. Ensure you are logged into Docker hub and have rights to push to kudobuilder.
 1. Tag repo with expected release `git tag -a v0.2.0 -m "v0.2.0"`  && push tag `git push --tags`.
 1. Invoke goreleaser `goreleaser --rm-dist`.
-1. Update the GH release with Release highlights and a changelog.
+1. Update the GH release with Release highlights and a changelog. There is a draft that contains categorized changes since the last release, use this as a template. It provides categories for highlights and breaking changes, the issues there should get a more detailed description.
```

Comment:
additionally... after using the draft for the release... delete the draft.

Reply:
I don't think it's necessary, as the release drafter would overwrite the draft with the new changelog on the next landed PR, but i've added a comment that the draft can be deleted afterwards.